### ISSUE TITLE
Error on compare

### ISSUE THREAD

**Issue by mnicely** [ionelmc/pytest-benchmark#173]
Running 
pytest 5.4.2
pytest-benchmark 3.2.3

I'm getting the following error on `pytest-benchmark compare 0001 0002`

```bash
/home/belt/anaconda3/envs/cusignal/bin/pytest-benchmark: 2: exec: /home/conda/feedstock_root/build_artifacts/pytest-benchmark_1585050181913/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/bin/python: not found
```

**Comment by ionelmc** [comment]
Looks like your installation is broken, either reinstall pytest-benchmark or recreate the whole env  - looks like the bin is broken.

PS. Workaround: `python $(which pytest-benchmark) compare 0001 0002`

**Comment by mnicely** [comment]
I was able to fix it by reverting to v3.2.2.

**Comment by ionelmc** [comment]
Can't be. CLI didn't change from 3.2.2 to 3.2.3. Upgrade it to 3.2.3 or reinstall. Pretty sure it will work after that.
